src/app/logs_db/bot_update.py

```
@@ -1,29 +1,99 @@
+
+    This function attempts to add the date_only column to both
+    log tables. If the column already exists, the ALTER TABLE
+    statement will fail silently.
+
+    Affected Tables:
+        - logs
+        - list_logs
+    """
     try:
         db_commit("ALTER TABLE logs ADD COLUMN date_only DATE")
     except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)